src/main/properties/datasets.properties

```diff
@@ -521,304 +649,416 @@ org.aksw.gerbil.datasets.definition.Masakha-Yoruba-Train.experimentType=RT2KB
 ### Masakha 2
 org.aksw.gerbil.datasets.masakha2DatasetConfiguration.datasetdir=${org.aksw.gerbil.DataPath}/datasets/masakha2
 ### Bambara language
+org.aksw.gerbil.datasets.definition.Masakha2-Bambara-Dev.group=MasakhaNER 2.0 Bambara
```

Comment:
This should only be `MasakhaNER 2`. The same holds for the other datasets with this name.

Reply:
Done